frontend/src/components/mine/MineHeader.js

```diff
@@ -249,6 +252,17 @@ export class MineHeader extends Component {
       </Menu>
     );
 
+    const mapRoute = this.props.mine.mine_location
+      ? route.MINE_HOME_PAGE.mapRoute(
+          null,
+          null,
+          null,
+          this.props.mine.mine_location.latitude,
+          this.props.mine.mine_location.longitude,
+          12
+        )
```

Comment:
This is far too many positional arguments. 3 is generally the acceptable max, imo. If you want to pass in more arguments, I'd recommend passing in an object so that the arguments can be passed in by name.

Reply:
done